src/core/contextManager.ts

```diff
@@ -12,10 +12,19 @@ type CoderContext = keyof typeof CONTEXT_DEFAULTS;
 export class ContextManager implements vscode.Disposable {
 	private readonly context = new Map<CoderContext, boolean>();
 
-	public constructor() {
-		(Object.keys(CONTEXT_DEFAULTS) as CoderContext[]).forEach((key) => {
+	public constructor(extensionContext: vscode.ExtensionContext) {
+		for (const key of Object.keys(CONTEXT_DEFAULTS) as CoderContext[]) {
 			this.set(key, CONTEXT_DEFAULTS[key]);
-		});
+		}
+		this.setInternalContexts(extensionContext);
+	}
+
+	private setInternalContexts(extensionContext: vscode.ExtensionContext): void {
+		vscode.commands.executeCommand(
```

Comment:
Just curious why this uses `vscode.commands.executeCommand` instead of `this.set`.  Seems like the main result is that `this.context` would not get updated, is that OK?